instrumentation/servlet/servlet-2.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v2_2/Servlet2HttpServerTracer.java

```diff
@@ -16,7 +16,7 @@ public static Servlet2HttpServerTracer tracer() {
 
   @Override
   protected String getInstrumentationName() {
-    return "io.opentelemetry.javaagent.servlet";
+    return "io.opentelemetry.javaagent.servlet22";
```

Comment:
```suggestion
    return "io.opentelemetry.javaagent.servlet-2.2";
```

Nit but we may as well use the same formatting as the artifact name for these

Reply:
seems reasonable to me!